metagpt/actions/rebuild_sequence_view.py

```diff
@@ -60,3 +304,77 @@ def _get_full_filename(root: str | Path, pathname: str | Path) -> Path | None:
             if str(i).endswith(postfix):
                 return i
         return None
+
+    @staticmethod
+    def parse_participant(mermaid_sequence_diagram: str) -> List[str]:
+        pattern = r"participant ([a-zA-Z\.0-9_]+)"
+        matches = re.findall(pattern, mermaid_sequence_diagram)
+        return matches
+
+    async def _search_new_participant(self, entry: SPO) -> str | None:
+        rows = await self.graph_db.select(subject=entry.subject, predicate=GraphKeyword.HAS_SEQUENCE_VIEW)
```

Comment:
Is it proper to move graph_db.select/insert into single func like `_get_class_detail`, so the db operations are in independent logic.

Reply:
Graph database operations have been encapsulated in `GraphRepository` and its subclasses. Here, `RebuildSequenceView` and `GraphRepository` have a composition relationship, so `GraphRepository`'s interface is directly called.